test/HW04Tests.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE NondecreasingIndentation #-}
```

Comment:
This shouldn't be needed if you format things properly (and it should be easy to forma things properly if you use formatter like fourmolu :slightly_smiling_face: )
```suggestion
```